toggle-dummy.js

```diff
@@ -1,8 +1,27 @@
-  $(function(){
-    var button =
-      $('<button type="button" onclick="toggleDummy()"> Show dummies</button>');
-    $("body").prepend(button);
-  });
-  function toggleDummy(){
-    $(".dummy").toggleClass("is-active")
+$(function(){
+  addButton()
```

Comment:
What do you think about an empty line between this function and the next conditional?